azure-mgmt-resources/src/main/java/com/azure/management/resources/AliasPathType.java

```diff
@@ -1,7 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-// 
+// Licensed under the MIT License.

```

Comment:
Should we apply this change to all existing files?

Reply:
We can get some script to do this. Maybe later when someone finished more important tasks...

Currently newly generated code will use this short style.